newIDE/app/src/Profile/ContributionsDetails.js

```diff
@@ -69,18 +70,24 @@ export const ExamplesAccordion = ({
         </Text>
       </AccordionHeader>
       <AccordionBody disableGutters>
-        <Column>
-          {examples.map(example => (
-            <ContributionLine
-              key={example.name}
-              shortDescription={example.shortDescription}
-              fullName={example.name}
-              previewIconUrl={
-                example.previewImageUrls ? example.previewImageUrls[0] : ''
-              }
-            />
-          ))}
-        </Column>
+        {examples.length === 0 && (
+          <EmptyMessage>You haven't contributed any examples</EmptyMessage>
+        )}
+
+        {examples.length !== 0 && (
```

Comment:
Use `<Trans>` to mark strings to be translated in JSX, and add a period at the end of the sentence.
Also nitpicking: prefer using the ternary to make it explicit this a choice between one or the other :) 

```suggestion
        {examples.length === 0 ? (
          <EmptyMessage><Trans>You haven't contributed any examples.</Trans></EmptyMessage>
        ) : (
```